tests/test_parse_warn.py

```
@@ -229,6 +229,48 @@ def test_pyi6_skip_set_filtered_even_when_delayed(self):
+        # derived requirement: prior to this test, only 2 of SKIP's ~14 non-collections/
+        # importlib entries (grp, posix) had any dedicated test exercising them through
+        # parse_warn_file -- the rest (pwd, fcntl, resource, readline, termios, tty, pty,
+        # crypt, spwd, nis, syslog, ossaudiodev) were only asserted present in the SKIP
+        # frozenset itself, never proven to actually filter when they appear in a real
+        # PyInstaller 6.x warn line. This test iterates the CURRENT SKIP set (any future
+        # addition is covered automatically, no separate registry to keep in sync) and
+        # proves each one is filtered in both the (conditional) and (delayed) forms real
+        # warn files actually use for these stdlib shims.
+        for mod in sorted(SKIP):
+            for qualifier in ("conditional", "delayed", "top-level"):
+                line = "missing module named {} - imported by app ({})".format(mod, qualifier)
```

Ruff (0.16.0)

[warning] Use f-string instead of `format` call

Convert to f-string

(UP032)